dynamic-config/config-apollo/src/main/java/org/apache/servicecomb/config/archaius/sources/ApolloConfigurationSourceImpl.java

```diff
@@ -49,11 +49,21 @@
 
   private List<WatchedUpdateListener> listeners = new CopyOnWriteArrayList<>();
 
+  private static final String APOLLO_CONFIG_URL_KEY = "apollo.config.serverUri";
+
   public ApolloConfigurationSourceImpl() {
   }
 
   private final UpdateHandler updateHandler = new UpdateHandler();
 
+  @Override
+  public boolean isValidSource(Configuration localConfiguration) {
+    if (localConfiguration.getProperty(APOLLO_CONFIG_URL_KEY) == null){
```

Comment:
It's better to write the Log here (to let the user know about which configure is wrong).

Reply:
Fixed.